config/manager/manager.yaml

```diff
@@ -29,11 +29,6 @@ spec:
         - --enable-leader-election
         image: controller:latest
         name: manager
-        resources:
-          limits:
-            memory: 100Mi
-          requests:
```

Comment:
We should likely leave requests in place.

Reply:
These are in line with CAPA, which when I debugged this same issue was having issues as well. 

In this case, the reason to remove the requests as well is that the pod will get the namespace default requests/limits

Reply:
I think ideally we wouldn't want to rely on namespace defaults here and should instead be providing a more targeted approximation of memory usage (under normal operations) for the requests. @ncdc thoughts?

Reply:
I was leaning towards removing any assumption of what a default scenario would be, given that really any scenario is going to be different. That said, I'm ok switching it back to what it was before, and only remove limits

Reply:
I tend to be torn on cpu requests and limits, since the typical answer to what you should use is "it depends". When we wrote Velero, we shipped with no requests or limits for a long time. After the community asked for some guidelines, the Velero team did some profiling to have at least an informed baseline. If it's acceptable, my recommendation would be:

1. Document that users should set requests and limits. If we don't have recommended baseline values today, we should say so in the docs, and indicate that we plan to provide that data in the future after performing profiling.
    1. I'm not sure if it makes sense to include arbitrary requests and limits in the yaml, or omit them
1. Perform said profiling
1. Update yaml with baselines. Make sure docs still indicate that these may need to be adjusted per deployment, based on the user's environment.

Reply:
I'll steal this comment to open a new issue

Reply:
My biggest concern with the omission of any requests is that it could easily lead to over-provisioning on a management cluster. But as @ncdc mentioned arbitrary values wouldn't be valuable there.

Reply:
I would rather default to deploying "full throttle" vs. artificially restricted based on kubebuilder defaults. Especially if we document ASAP that you _should_ set requests and limits, and our data is forthcoming.

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/1677